src/mono/netcore/System.Private.CoreLib/src/System/Reflection/Emit/ILGenerator.Mono.cs

```diff
@@ -1055,11 +1053,11 @@ internal void label_fixup(MethodBase mb)
         }
 
         // Used by DynamicILGenerator and MethodBuilder.SetMethodBody
-        internal void SetCode(byte[] code, int max_stack)
+        internal void SetCode(byte[]? code, int max_stack)
         {
             // Make a copy to avoid possible security problems
-            this.code = (byte[])code.Clone();
-            this.code_len = code.Length;
+            this.code = code != null ? (byte[])code.Clone() : Array.Empty<byte>();
```

Comment:
This is a functional change to match what coreclr does rather than, you know, null ref'ing :)